components/builder-web/app/actions/builds.ts

```diff
@@ -59,11 +59,13 @@ export function fetchBuildLog(id: string, token: string, start = 0) {
       .then((data) => {
           dispatch(populateBuildLog(data));
 
-          if (data["is_complete"] && start !== 0) {
+          let complete = data["is_complete"];
+
+          if (complete && data["start"] !== 0) {
```

Comment:
I'm not quite following what this is doing... if the last chunk you got was the last one of the log, fetch the entire log in 5 seconds?